<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>828</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>武易高速道路运输监测数据展示系统</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="userMenu">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>用户</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actUser_current"/>
    <addaction name="separator"/>
    <addaction name="actUser_manage"/>
   </widget>
   <widget class="QMenu" name="pageMenu">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>页面</string>
    </property>
    <addaction name="actMenu_home"/>
    <addaction name="separator"/>
    <addaction name="actMenu_place"/>
    <addaction name="actMenu_device"/>
    <addaction name="separator"/>
    <addaction name="actMenu_carData"/>
    <addaction name="actMenu_personData"/>
   </widget>
   <widget class="QMenu" name="aboutMenu">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actAbout_app"/>
    <addaction name="actAbout_qt"/>
   </widget>
   <addaction name="userMenu"/>
   <addaction name="pageMenu"/>
   <addaction name="aboutMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actPage_home"/>
   <addaction name="separator"/>
   <addaction name="actPage_place"/>
   <addaction name="actPage_device"/>
   <addaction name="separator"/>
   <addaction name="actPage_carData"/>
   <addaction name="actPage_personData"/>
  </widget>
  <action name="actPage_home">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;打开首页&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看系统总体统计信息</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actPage_device">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/device.png</normaloff>:/icons/device.png</iconset>
   </property>
   <property name="text">
    <string>监测设备</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;打开设备管理界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理监测设备</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actPage_carData">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/car.png</normaloff>:/icons/car.png</iconset>
   </property>
   <property name="text">
    <string>车流数据</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;打开车流数据界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理车流数据</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actPage_personData">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/person.png</normaloff>:/icons/person.png</iconset>
   </property>
   <property name="text">
    <string>人流数据</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;打开人流数据界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理人流数据</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actUser_current">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
   </property>
   <property name="text">
    <string>当前用户</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;打开“当前用户”界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和修改当前用户的信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actPage_place">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/place.png</normaloff>:/icons/place.png</iconset>
   </property>
   <property name="text">
    <string>监测点</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;打开监测点管理界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理监测点</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actMenu_home">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;首页&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看系统总体统计信息</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actMenu_device">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>监测设备</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;设备管理界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理监测设备</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actMenu_place">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>监测点</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;监测点管理界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理监测点</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actMenu_carData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>车流数据</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;车流数据界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理车流数据</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actMenu_personData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>人流数据</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;人流数据界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和管理人流数据</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actUser_manage">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/user_manage.png</normaloff>:/icons/user_manage.png</iconset>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;打开用户管理界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>查看和修改观察员的信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAbout_app">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>关于本软件</string>
   </property>
   <property name="statusTip">
    <string>查看本软件的相关信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAbout_qt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/qt.png</normaloff>:/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
   <property name="statusTip">
    <string>查看Qt的相关信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
